frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -164,6 +165,10 @@ export const surveysLogic = kea<surveysLogicType>([
             (s) => [s.hasAvailableFeature],
             (hasAvailableFeature) => hasAvailableFeature(AvailableFeature.SURVEYS_STYLING),
         ],
+        surveysConfigAvailable: [
+            (s) => [s.hasAvailableFeature],
+            (hasAvailableFeature) => hasAvailableFeature(AvailableFeature.SURVEYS_STYLING),
```

Comment:
yeah, at this point I'd call it `styling` over `settings`, but that might be a nitpick.